kedro/framework/cli/project.py

```diff
@@ -218,17 +246,33 @@ def build_docs(metadata: ProjectMetadata, open_docs):
 
 
 @forward_command(project_group, name="build-reqs")
+@click.option(
+    "--input-file",
+    "input_file",
+    multiple=False,
+    default=False,
```

Comment:
I don't think `default` is needed for options - I think they're only relevant for flags.